experimental/devices/inky/inky.go

```diff
@@ -38,9 +32,9 @@ const (
 
 var borderColor = map[Color]byte{
 	Black:  0x00,
-	Red:    0x33,
+	Red:    0x73,
```

Comment:
That's weird, what happened?

Reply:
This one was changed upstream in https://github.com/pimoroni/inky/commit/4769114be193d44215ae64236e74f25f7b485c7a for https://github.com/pimoroni/inky/issues/12